benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -117,9 +117,13 @@ def prepare_op_inputs(op, arg_params):
 
     # 3d tensor is needed by following ops
     ops_3d = ['CTCLoss', 'ctc_loss']
-
+    
```

Comment:
nit: fix indent
if its the only change needed don't do it in this PR to prevent CI reruns

Reply:
Actually you need to rebase anyway so fix it in this PR 😛